3.7/upgrading-starter.md

```diff
@@ -128,23 +128,50 @@ max      29419  3684  0 11:46 pts/1    00:00:00 arangodb --starter.data-dir=./db
 max      29504  3695  0 11:46 pts/2    00:00:00 arangodb --starter.data-dir=./db2 --starter.join 127.0.0.1
 max      29513  3898  0 11:46 pts/4    00:00:00 arangodb --starter.data-dir=./db3 --starter.join 127.0.0.1
 ```
+We can use `pstree` to inspect the arangodb-server instances launched by one of these starters:
+
+```bash
+pstree -Tp 3511800 
+arangodb(29419)─┬─arangod(30201)
+                ├─arangod(30202)
+                └─arangod(30217)
+```
 
 ### Restart the _Starter_
 
 When using a supervisor like _SystemD_, this will happen automatically. In case
 the _Starter_ was initiated manually, the _arangodb_ processes have to be restarted
 manually with the same command that has been used before.
 
+Inspect which processes belong one starter using the `pstree` command from above.
+
 If you are using the `.tar.gz` distribution (only available from v3.4.0),
 your new version of the executable might be located in a
 different directory. Make sure that you now start the new _Starter_
 executable (`bin/arangodb`) in the new installation place. If you are
 using a supervisor like _SystemD_, you might have to adjust the path to
-the executable in the service description to the new location. Do this
-before you `kill -9` the _Starter_ or else the old version will be
+the executable in the service description to the new location. Also make sure 
+that the unit file contains
+[`KillMode=process`](https://www.freedesktop.org/software/systemd/man/systemd.kill.html#KillMode=)
+else this procedure will fail.
```

Comment:
```suggestion
Otherwise this procedure will fail, because the default is `KillMode=control-group` which will kill
the _arangod_ processes that were started by an _arangodb_ starter process.
```